day_02.md

```diff
@@ -38,59 +38,60 @@
+> In computer programming, a statement is a syntactic unit of an imperative programming language that expresses some action to be carried out. A program written in such a language is formed by a sequence of one or more statements. A statement may have internal components (e.g., [expressions](<https://en.wikipedia.org/wiki/Expression_(computer_science)>)).
 >
-> Source : [Wikipedia](https://en.wikipedia.org/wiki/Statement_(computer_science))
+> Source : [Wikipedia](<https://en.wikipedia.org/wiki/Statement_(computer_science)>)
 
 Ok that's hard, let's have some help from Kyle Simpson
+
 > In a computer language, a group of words, numbers, and operators that performs a specific task is a statement.
 >
 > Source: [YDKJS - Up & Going - 1st edition](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/up%20%26%20going/ch1.md#statements)
 
-Now that's better!!
+Now that's better!
 
-JavaScript is quite an expressive language with a nice set of statement categories on it's toolkit.
+JavaScript is quite an expressive language with a nice set of statement categories in it's toolkit.
 
 > The appearance of statements shapes the look of programs. Programming languages are characterized by the type of statements they use (e.g. the curly brace language family). Many statements are introduced by identifiers like `if`, `while` or `repeat`. Often statement keywords are reserved such that they cannot be used as names of variables or functions. Imperative languages typically use special syntax for each statement, which looks quite different from function calls.
 >
-> Source: [Wikipedia](https://en.wikipedia.org/wiki/Statement_(computer_science)#Syntax)
+> Source: [Wikipedia](<https://en.wikipedia.org/wiki/Statement_(computer_science)#Syntax>)
 
 Here some categories:
 
 - [Control Flow](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements#Control_flow)
-  > Determinate the flow of a program
+  > Determine the flow of a program
 - [Declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements#Declarations)
-  > Describes an [identifier/token](https://en.wikipedia.org/wiki/Identifier#In_computer_languages) ( A lexical token or simply token is a string with an assigned and thus identified meaning [*](https://en.wikipedia.org/wiki/Lexical_analysis#Token) ) and optionally initializing it to a value.
+  > Describes an [identifier/token](https://en.wikipedia.org/wiki/Identifier#In_computer_languages) ( A lexical token or simply token is a string with an assigned and thus identified meaning [\*](https://en.wikipedia.org/wiki/Lexical_analysis#Token) ) and optionally initializing it to a value.
```

Comment:
backslash for [\*] is not required, maybe it's trying to lint it with the wrong linter?